example_rails6/babel.config.js

```diff
@@ -56,9 +56,7 @@ module.exports = function(api) {
       [
         '@babel/plugin-transform-runtime',
         {
-          helpers: false,
-          regenerator: true,
-          corejs: false
+          helpers: false
```

Comment:
Background on this change?

Reply:
everything in the `example_rails6` folder is generated